scripts/hudson/narayana.sh

```diff
@@ -349,25 +349,8 @@ function clone_as {
 
   cd ${WORKSPACE}
   if [ -d jboss-as ]; then
-    echo "Updating existing checkout of WildFly"
+    echo "Using existing checkout of WildFly. If a fresh build should be used, delete the folder ${WORKSPACE}/jboss-as"
     cd jboss-as
-
-    git remote | grep upstream
-    if [ $? -ne 0 ]; then
-      git remote add upstream https://github.com/wildfly/wildfly.git
-    fi
-    #Abort any partially complete rebase
-    git rebase --abort
-    git checkout 5_BRANCH
```

Comment:
I am not sure why it used to checkout the 5_BRANCH, but I think it is related to the Narayana version. I suppose now we should have a 6_BRANCH and set it as default.  WDYT @jmfinelli ?

Reply:
I think that the `main` branch should be used (for the current version of Narayana) and a maintenance branch (e.g. `5_BRANCH`) should be created when needed.